--- fstab.rk30board.bootmode.emmc	2016-10-12 17:36:34.698989006 +0200
+++ /usr/src/malaysk-rom/dupdate.img.dump/Image/boot.img.dump/fstab.rk30board.bootmode.emmc	2016-10-12 17:52:39.471519030 +0200
@@ -5,12 +5,12 @@
 /dev/block/platform/emmc/by-name/system         /system             ext4      ro,noatime,nodiratime,noauto_da_alloc                            wait,resize
 /dev/block/platform/emmc/by-name/cache          /cache              ext4      noatime,nodiratime,nosuid,nodev,noauto_da_alloc,discard          wait
 /dev/block/platform/emmc/by-name/metadata       /metadata           ext4      noatime,nodiratime,nosuid,nodev,noauto_da_alloc,discard          wait
-/dev/block/platform/emmc/by-name/userdata       /data               ext4      noatime,nodiratime,nosuid,nodev,noauto_da_alloc,discard          wait,check,encryptable=/metadata/key_file
+/dev/block/platform/emmc/by-name/userdata       /data               ext4      noatime,nodiratime,nosuid,nodev,noauto_da_alloc,discard          wait,encryptable=/metadata/key_file
 # /dev/block/platform/emmc/by-name/radical_update /radical_update     ext4      noatime,nodiratime,nosuid,nodev,noauto_da_alloc,discard          wait
 
 /dev/block/platform/emmc/by-name/oem       /mnt/oem           ext4      noatime,nodiratime,nosuid,nodev,noauto_da_alloc,discard          wait
 
-/devices/platform/emmc/mmc_host/mmc          /mnt/internal_sd     vfat      defaults        voldmanaged=internal_sd:12,noemulatedsd
+#/devices/platform/emmc/mmc_host/mmc          /mnt/internal_sd     vfat      defaults        voldmanaged=internal_sd:12,noemulatedsd
 /devices/platform/rk29_sdmmc.0/mmc_host/mmc          /mnt/external_sd     vfat      defaults        voldmanaged=external_sd:auto
 /devices/platform/rk29_sdmmc.1/mmc_host/mmc    /mnt/external_sd2     vfat     defaults        voldmanaged=external_sd2:auto
 
